amr-wind/atmospheric_boundary_layer/convective_abl_nrel5mw/README.md

```diff
@@ -25,17 +35,144 @@ Input files are in the [input_files](input_files) directory.  There are two stag
+
+ - $u^*$ = 0.468 m/s
+ - $z_i$ = 803.133 m
+ - Obukhov Length = -2447.446 m (computed at z=27 m)  
+
+
+### Flow Visualizations
 
-- Instantaneous hub-height wind speed contours
+An instantaneous hub-height visualization of the horizontal velocity at $t=15,000$s is included below. 
 ![HH WS contour](results/XYdomain_15000.png)
 
-- Averaged wind profiles from a virtual metmast at the turbine location
-![Met mast average](results/avgmetmast_0600.png)
+### Horizontal Profiles
+
+Horizontally averaged profiles are computed in the [AVG_horiz_profiles.ipynb](postprocessing/AVG_horiz_profiles.ipynb) Jupyter notebook are shown below. 
+
+**Note**: The python routines for computing horizontal profiles from the statistics file(s) generated by AMR-Wind rely on importing the [postproamrwindabl](https://github.com/Exawind/amr-wind-frontend/blob/main/postproamrwindabl.py) module from the [AMR-Wind front end](https://github.com/Exawind/amr-wind-frontend) library. It is not necessary to import the entire AMR-Wind front end library to compute the horizontal profiles.  If necessary, download the module files and edit the lines in the python code which define `postproamrwinddir` to include the location of that module:
+```python
+# Location of postproamrwindabl module inside the amr-wind-frontend
+postproamrwinddir = '~/src/amr-wind-frontend/'
+import sys, os, shutil, io
+if postproamrwinddir not in sys.path:
+    sys.path.append(postproamrwinddir)
+```
+
+#### Horizontal velocity:
+![Uhoriz_Profile](postprocessing/figures/AVG_horiz_profiles_Uhoriz.png)
+
+#### Temperature:
+![T_Profile](postprocessing/figures/AVG_horiz_profiles_T.png)
+
+#### Wind Direction:
+![winddir_profile](postprocessing/figures/AVG_horiz_profiles_WindDir.png)
 
-- Horizontally averaged wind profiles [AVG_horiz_profiles.ipynb](postprocessing/AVG_horiz_profiles.ipynb)
+#### Turbulence Intensity (TKE):
+![TI_profile](postprocessing/figures/AVG_horiz_profiles_TI.png)
+
+#### Resolved Reynolds stresses:
+![uw_profile](postprocessing/figures/AVG_horiz_profiles_RE.png)
+
+#### Resolved Temperature Fluxes:
+![vw_profile](postprocessing/figures/AVG_horiz_profiles_uT.png)
+
+### Metmast Profiles
+
+Averaged wind profiles from a virtual metmast at the turbine location are computed in the [AVG_metmast.ipynb](postprocessing/AVG_metmast.ipynb) Jupyter notebook and are shown below.
+
+![Met mast average](results/avgmetmast_0600.png)
 
-- Time frequency wind spectra [ABLSpectra.ipynb](postprocessing/ABLSpectra.ipynb)
+**Note**: The path to the [AMR-Wind front end](https://github.com/Exawind/amr-wind-frontend) library must be provided in the [ABL_metmast.ipynb](postprocessing/ABL_metmast.ipynb) Jupyter notebook.  If necessary, download the library and edit the lines which define `amrwindfedirs` to include any locations of that library, e.g.,
```

Comment:
Getting a file not found for this notebook as well.

Reply:
Should be fixed.